File: back/Models/Rentee.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace back.Models
{
    public class Rentee
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Username { get; set; }
        public decimal Password { get; set; }
        public string PhoneNumber { get; set; }
        public string Email { get; set; }
        public string ProfilePictureUrl { get; set; }

    }
}

File: front/src/app/models/Rentee.ts
export class Rentee{
    id:string;
    name: string;
    username: string;
    email: string;
    password: string;
    phoneNumber: string;
    profilePictureUrl: string;

    constructor(name, username, email, pass, phone, picUrl) {
       this.name=name;
       this.username=username;
       this.email=email;
       this.password=pass;
       this.phoneNumber=phone;
       this.profilePictureUrl=picUrl;
    }
  
}